File: Day10/other/Person.java
package other;

public class Person {
	protected String name;
	protected int age;
	
	public Person(String name, int age) {
		this.name = name;
		this.age = age;
	}
	
	public void eat(String food) {
		String msg = "%s(이)가 %s(을)를 먹는다\n";
		System.out.printf(msg, name, food);
	}
	
	public void showInfo() {
		String msg = "%s (%d세)\n";
		System.out.printf(msg, name, age);
	}
	
	public String getName() {
		return name;
	}
}

File: Day37/js/person.js
let per = {
    name : "홍길동",
    age : 23,

    show : function() {
        // 이름 나이를 문자열로 반환
        let msg = `${this.name} : ${this.age}세`;
        return msg;
    },

    eat : function(food) {
        // 'name'이 'food'를 먹는다를 문자열로 반환
        let msg = `${this.name}(이)가 ${food}(을)를 먹는다`;
        return msg;
    }
}